client/src/components/application/ResponsiveDrawer/list.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemText from '@material-ui/core/ListItemText';
+import { loadCSS } from 'fg-loadcss';
+import Icon from '@material-ui/core/Icon';
+import PropTypes from 'prop-types';
+import useStyles from './style';
+
+const DrawerList = ({ content, isFinished }) => {
+  const classes = useStyles();
+  React.useEffect(() => {
+    loadCSS(
```

Comment:
Could you load this css by putting a new link tag in the head, in your index.html file, somewhere near here?
https://github.com/GSG-G8/applicants-system/blob/57892dbffa1debd0c7bc3e2e42c4cfcc91cee3cd/client/public/index.html#L8

That would be a little simpler than this design.  Also, then the css would already be downloaded and ready by the time you are ready to show this component.  With useEffect loading the css, it won't begin to download that external css until this component is mounted, which might mean that the user sees a flash of unstyled content while they wait for the css to download and then be applied to the page.